packages/gas-fee-controller/src/gas-util.ts

```diff
@@ -191,3 +191,30 @@ export function calculateTimeEstimate(
     upperTimeBound,
   };
 }
+
+/**
+ * Build an infura auth token from the given API key and secret.
+ *
+ * @param infuraAPIKey - The Infura API key.
+ * @returns The base64 encoded auth token.
+ */
+export function buildInfuraAuthToken(infuraAPIKey: string) {
```

Comment:
If we build the token with the `fetchX` methods, this shouldn't need to be exported.

Reply:
Done in https://github.com/MetaMask/core/pull/4068/commits/cd6bb24d1fb5e5215ff46f85b3e820c1064a910e